tests/Unit/Routes/StoreRoutesTest.php

```diff
@@ -163,6 +163,30 @@ public function testUpdateRouteGate()
         ;
     }
 
+    /** @test */
+    public function testVoucherManageRouteGate()
+    {
+        // Create a random registration with our centre.
+        $registration = factory(Registration::class)->create([
+            "centre_id" => $this->centre->id,
+        ]);
+
+        $route = URL::route('store.registration.voucher-manager', [ 'registration' => $registration->id ]);
+
+        Auth::logout();
+        // You cannot get there logged out.
+        $this->visit($route)
+            ->seePageIs(URL::route('store.login'))
+            ->assertResponseStatus(200)
+        ;
+        // You can get there logged in.
```

Comment:
Can you get there if you are logged in as a centre user from another centre?

Reply:
I'll add a test for that.

Reply:
The test bellow here seems to test that